allennlp/common/util.py

```diff
@@ -220,8 +220,15 @@ def prepare_global_logging(serialization_dir: str, file_friendly_logging: bool)
         The directory to stream logs to.
     file_friendly_logging : ``bool``, required.
         Whether logs should clean the output to prevent carridge returns
-        (used to update progress bars on a single terminal line).
+        (used to update progress bars on a single terminal line). This
+        option is only used if you are running in an environment with a terminal.
     """
+
+    # If we don't have a terminal as stdout,
+    # force tqdm to be nicer.
+    if not sys.stdout.isatty():
+        file_friendly_logging = True
```

Comment:
is there ever a case where you'd want file_friendly_logging in a terminal?

that is, with this change can we just get rid of the file_friendly_logging config parameter everywhere?

Reply:
I use file_friendly_logging with a terminal.